### ISSUE TITLE
BUG: ntpdate return code 1

### ISSUE THREAD

**Issue by cmstew** [hestiacp/hestiacp#554]
### In a few words, please describe the issue that you're experiencing:

Every morning I receive an email from cron that says: `run-parts: /etc/cron.daily/ntpdate exited with return code 1`.

I've tried running the command inside manually, without the silent switch and get this: `the NTP socket is in use, exiting`.

I think the problem is that there is already an ntp service running.

### Expected behavior:
Not email me every day, and to not run a potentially unnecessary piece of code.

### Operating system distribution and release:
Debian 9.7

### Which version of Hestia Control Panel is currently installed?
1.0.4

### Which branch are you using?
release

### When did this issue occur? After a clean installation, or after an upgrade?
after clean

### Which of the following web server configurations are you using?

- Nginx + PHP-FPM

### Which of the following server roles are installed on your server?
- Mail (Dovecot + Exim)
- MySQL (MariaDB)

**Comment by ScIT-Raphael** [comment]
Hestia does install the ntpdate by default, but does not set any additional configurations. I would suggest to google that message, have seen a lot of entries with similar logs.

**Comment by cmstew** [comment]
Hey @ScIT-Raphael 

Coming back to this all this time later. I found that the reason I got these errors was because there is a ntp service that is installed and enabled by default on the version of Debian that I use. While it's running, it blocks the ntp port and doesn't allow the cron job to run.

Maybe we should consider checking for an ntp service and disabling it to prevent conflict on fresh installs?
